src/substrait/__init__.py

```diff
@@ -1,4 +0,0 @@
-try:
```

Comment:
Hmm, this gets overwritten by ./gen_proto.sh when we don't use a subdirectory within the package.

Reply:
hm, I was looking to see if we could move this somewhere, but I'm suddenly unsure why this is getting overwritten if it should be putting files into a `proto` directory.

Reply:
I think I prefer to keep the generated files in `substrait.proto.<namespace>`